docs/kuneiform/ddl.md

```diff
@@ -73,22 +88,22 @@ table posts {
     userid int notnull,
     username text notnull,
     content text notnull,
-    foreign_key (userid) references users(id) on_delete cascade
-    foreign_key (username) references users(name) on_update cascade
+    foreign key (userid) references users(id) on delete cascade
+    foreign key (username) references users(name) on update cascade
 }
 ```
 
 Below are a list of operations that available for the foreign key declaration:
 
-- ```no_action``` No operation is triggered.
+- ```no action``` No operation is triggered.
 - ```restrict``` Other calls to the other refrenced column will fail to update the column.
-- ```set_null``` The column is set to null.
-- ```set_default``` The column is set to the column's default value (if specified in the table declaration).
+- ```set null``` The column is set to null.
+- ```set default``` The column is set to the column's default value (if specified in the table declaration).
 - ```cascade``` The change in the reference column cascades to the specified column.  In the case of a deletion, it will delete the entire record.
 
 ## Indexing in Tables
 
-The Kuneiform Language not only allows column declarations, but also the creation of indexes on one or more columns within your table. Kuneiform supports standard SQLite B-tree indexes. For more details on SQLite indexes, please refer to [this tutorial](<https://www.sqlitetutorial.net/sqlite-index/>).
+The Kuneiform Language not only allows column declarations, but also the creation of indexes on one or more columns within your table. Kuneiform supports standard B-tree indexes.
 
 ## Index Declaration
```

Comment:
I think "Index Declaration" should be contained within "Indexing in Tables" (as opposed to at the same level).

```suggestion
### Index Declaration
```